docs/topics/tour/kotlin-tour-control-flow.md

```diff
@@ -162,6 +189,85 @@ You can also do the same with `Char` ranges:
 * `'a'..'d'` is equivalent to `'a', 'b', 'c', 'd'`
 * `'z' downTo 's' step 2` is equivalent to `'z', 'x', 'v', 't'`
 
+## Conditional expressions and ranges practice
+
+### Exercise 1 {initial-collapse-state="collapsed" id="conditional-expressions-exercise-1"}
+
+Create a simple game where you win if throwing two dice results in the same number. Use `if` to print `You win :)`
+if the dice match or `You lose :(` otherwise.
+
+<deflist collapsible="true">
+    <def title="Hint">
+        Use the <a href="operator-overloading.md#equality-and-inequality-operators"> equality operator </a>  (<code>==</code>) to compare the dice results. 
+    </def>
+</deflist>
+
+|---|---|
```

Comment:
I'm a bit confused by this syntax, I don't see it changing anything on staging
Do two code samples actually go into columns in a table
Same for exercise 2

Reply:
It's a way of introducing a line break so that the formatting looks nicer.